assets/constants.js

```diff
@@ -441,5 +441,29 @@ var constants = {
 
   "WHITELISTED_COLLECTION_IDS_FOR_SAVING_GUEST_PROGRESS": [],
 
-  "FEEDBACK_SUBJECT_MAX_CHAR_LIMIT": 50
+  "FEEDBACK_SUBJECT_MAX_CHAR_LIMIT": 50,
+
+  "ACTIVITY_STATUS_PRIVATE": "private",
+  "ACTIVITY_STATUS_PUBLIC": "public",
+
+  "//": "Committer id for system actions.",
+  "SYSTEM_COMMITTER_ID": "admin",
```

Comment:
Is this actually used in the frontend at all? If not, we should keep it in feconf and not move it to constants, I think...

Ditto for other constants that aren't used in the frontend.

Reply:
Also, for constants related to each other, write some kind of backend test that (say) extracts SYSTEM_USER_IDS from constants.js and SYSTEM_COMMITTER_ID + MIGRATION_BOT_USERNAME from feconf.py, and checks that both of these match.

Reply:
Done.